### PR TITLE
:sparkles: add status tags to ui (view only)

### PR DESCRIPTION
**Pull request by MrKrisKrisu**
Anyone some font-awesome icon ideas?

Screenshot 2023-11-05 at 09 58 59

### CODE REVIEW

**Comment by vainamov** [comment]
- `<i class="fa-solid fa-ticket-simple"></i>` for the ticket type?
- `<i class="fa-solid fa-left-right"></i>` for the carriage number?
- `<i class="fa-solid fa-couch"></i>` instead of chair for the seat?

File: resources/views/status.blade.php [hunk 1]
```diff
@@ -42,6 +42,29 @@
                         {{$status->trainCheckin->HafasTrip->last_refreshed->diffForHumans()}}
                     </small>
                 @endif
+
+                @foreach($status->tags as $tag)
+                    @can('view', $tag)
+                        <span class="badge bg-trwl">
+                            @if($tag->key === 'trwl:seat')
```
Comment (HerrLevin) [inline resources/views/status.blade.php]: Maybe add an Enum with `\App\Interfaces\IconEnumInterface` to keep all icons at one central point and to reduce clutter?

**Comment by MrKrisKrisu** [comment]
> * `<i class="fa-solid fa-ticket-simple"></i>` for the ticket type?

Hmm, I'm not sure - I like the QR code for the ticket.

> * `<i class="fa-solid fa-left-right"></i>` for the carriage number?

I don't understand, how arrows represents the carriage number. :c 

> * `<i class="fa-solid fa-couch"></i>` instead of chair for the seat?

I like the couch instead of the seat, thanks. :)

**Comment by vainamov** [comment]
> Hmm, I'm not sure - I like the QR code for the ticket.

Fair enough :D

> I don't understand, how arrows represents the carriage number. :c

Now that I think about it, I wouldn't either. When I chose it, I thought it ever-so-slightly resembled a train from the side.

But considering the style of the tags, I don't think icons are necessary for each of them. So I would rather spare an icon than pick a less-suitable one.

**Comment by HerrLevin** [comment]
Maybe `<i class="fa-solid fa-van-shuttle"></i>` for the carriage number?
`<i class="fa-solid fa-ticket"></i>` for the class?

**Review by HerrLevin — APPROVED** [review]

**Pull request merged**